docs/ota-client-guide/modules/ROOT/pages/add-meta-updater-to-vendors-sdk.adoc

```diff
@@ -0,0 +1,87 @@
+= Add meta-updater features to the vendor's SDK
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+
+To avoid conflicts and bug classes, we require the use of `usrmerge`, even on systems that do not use systemd. In the https://www.yoctoproject.org/docs/2.7/mega-manual/mega-manual.html[Yocto] ecosystem, it is a general best practice to write `do_install()` functions that install to the variable `\{bindir}`, but some recipes still hard-code the location of `/bin`.

```

Comment:
```suggestion
To avoid conflicts and certain classes of bugs, we require the use of `usrmerge`, even on systems that do not use systemd. In the https://www.yoctoproject.org/docs/2.7/mega-manual/mega-manual.html[Yocto] ecosystem, it is a general best practice to write `do_install()` functions that install to the variable `\{bindir}`, but some recipes still hard-code the location of `/bin`.

```